### PR TITLE
ACTUALLY fixes Battle Royale

### PR DESCRIPTION
**Pull request by ynot01**
fuck

### CODE REVIEW

**Comment by ThatLing** [comment]
and i said virGINS

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Use TRUE/FALSE

Call parent

**Comment by Moja10** [comment]
PROMOTE THIS CHAD

File: yogstation/code/game/gamemodes/battle_royale/battleroyale.dm [hunk 1]
```diff
@@ -25,15 +25,10 @@ GLOBAL_LIST_EMPTY(battleroyale_players) //reduce iteration cost
 	var/mob/living/winner // Holds the wiener of the victory royale battle fortnight.
 	title_icon = "ss13"
 
-	var/list/possible_virgins = list()
-
 /datum/game_mode/fortnite/can_start()
 	if(!..())
 		return 0
```
Comment (alexkar598) [inline yogstation/code/game/gamemodes/battle_royale/battleroyale.dm]: Return FALSE

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: yogstation/code/game/gamemodes/battle_royale/battleroyale.dm (hunk 1 above)
Comment (ynot01) [inline yogstation/code/game/gamemodes/battle_royale/battleroyale.dm]:
```suggestion
		return FALSE
```

Comment (ynot01) [reply yogstation/code/game/gamemodes/battle_royale/battleroyale.dm]: im sorry hc please dont spank me

**Review by Amelia0010 — APPROVED** [review]

**Review by alexkar598 — DISMISSED** [review]
Actually looking over the changes myself, the first PR was bad and shouldnt of been merged at all, theres no need for an overridden can_start proc as all logic is already done at the parent /datum/game_mode/proc/can_start

tldr delete the entire proc it shouldnt exist

**Review by fluffe9911 — APPROVED** [review]
ok
